vagrant/ansible/roles/sit.cephfs/templates/smb_share.conf.j2

```diff
@@ -1,6 +1,9 @@
 [{{ name }}]
-comment = For samba share of volume {{ path }}
-vfs objects = acl_xattr
+comment = For samba share of subvolume {{ name }}
+vfs objects = acl_xattr{% if config.be.variant == 'vfs' %} ceph
```

Comment:
Minor style nitpick. Having an if block start in the middle of this line but continue over multiple lines looks really awkward to me. Perhaps just an if-else-endif would read better?

Reply:
I'll try to make it prettier.